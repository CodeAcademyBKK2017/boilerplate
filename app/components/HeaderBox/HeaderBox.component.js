import React, {Component} from 'react';
import style from './HeaderBox.component.style';
import {
  Text,
  View
} from 'react-native';

export default class TextArea extends Component {
  render () {
    return (
      <View>
        <View style={style.viewStyle}>
          <Text style={style.textStyle}>Start taking notes.</Text>
        </View>
        <View style={style.divide}/>
      </View>
    );
  }
}