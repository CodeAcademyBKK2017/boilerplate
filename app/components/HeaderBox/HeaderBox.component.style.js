import {StyleSheet} from 'react-native';

export default StyleSheet.create({
  viewStyle: {
    width: '100%',
    height: 60,
    paddingTop: 15,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: 'white'
  },
  textStyle: {
    fontWeight: 'bold',
    fontSize: 18
  },
  divide: {
    height: 1,
    width: '100%',
    backgroundColor: '#E0E0E0'
  }
});
