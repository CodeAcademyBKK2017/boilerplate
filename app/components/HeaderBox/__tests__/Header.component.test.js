import 'react-native';
import HeaderBox from '../HeaderBox.component';
import React from 'react';
import renderer from 'react-test-renderer';

describe('HeaderBox', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <HeaderBox />
    );
    expect(tree).toBeDefined();
  });

  it('HeaderBox: Should return TextArea', () => { // example to test class methods
    const tree = renderer.create(<HeaderBox />);
    expect(tree).toMatchSnapshot();
  });
});