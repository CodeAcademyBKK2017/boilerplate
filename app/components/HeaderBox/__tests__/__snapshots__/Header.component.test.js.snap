// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderBox HeaderBox: Should return TextArea 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "height": 60,
        "justifyContent": "center",
        "paddingTop": 15,
        "width": "100%",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 18,
          "fontWeight": "bold",
        }
      }
    >
      Start taking notes.
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#E0E0E0",
        "height": 1,
        "width": "100%",
      }
    }
  />
</View>
`;
